src/app/pages/TokenDashboardPage/NFTLinks.tsx

```diff
@@ -72,7 +72,7 @@ export const NFTOwnerLink: FC<NFTOwnerLinkProps> = ({ scope, owner }) => {
   const { t } = useTranslation()
 
   return (
-    <Typography sx={{ display: 'flex', whiteSpace: 'initial' }}>
+    <Typography sx={{ display: 'flex', whiteSpace: 'initial', pt: '4px', pb: '1px' }}>
```

Comment:
uh lets add a comment to each of these workarounds

including https://github.com/oasisprotocol/explorer/pull/1997/files#diff-ceb240d495577cc802f3d0448e0f0e41f397ad13b17c4420e2c20f232375c0d6R232